tbx/project_styleguide/templates/patterns/molecules/streamfield/blocks/showcase_block.yaml

```diff
@@ -0,0 +1,24 @@
+context:
+  value:
+    title: Our Services
+    showcase_paragraphs:
+      - page: '#'
+        heading: Sponsor a Wagtail feature or enhancement
+        summary: Join the likes of Mozilla, Google, Motley Fool and YouGov and sponsor enhancements to Wagtail.
+      - heading: Get expert developer support for your team
+        summary: Get the reassurance of having Wagtail developers on hand, just in case. Oh, and all profit supports the Wagtail project.
+      - page: '#'
+        heading: Supercharge your output with staff aug
+        summary: Our experienced Wagtail developers will join your team to add specialist capability and capacity
+      - heading: Outsourced Wagtail builds
+        summary: Everything from strategy, SEO, analytics, design, development, systems integration, devops and more.
+      - heading: Wagtail developer training
+        summary: Our training programme to build your understanding and skills in essential Django and Wagtail CMS practices.
+      - page: '#'
+        heading: 24/7 application support & hosting
+        summary: 24/7 application support and scalable, performant and resilient cloud hosting optimised for Wagtail.
+
+tags:
+  pageurl:
+    item.page:
+      raw: '#'
```

Comment:
Could you add in a `showcase_block.md` file and add some details about how these are expected to be used? We would not expect them to be used in practice with some items as links and some as not within the same showcase set - and that it is only done here as a demonstration. You also should update the editor guidelines to give details on this one.

Reply:
Added in https://github.com/torchbox/torchbox.com/pull/50/commits/6d552f4975d77a5da0659f2d0dc7556fa8246422 and also added a 'Streamfields' section in the editor guidelines

Reply:
also added an extra sentence in the editor's guidelines to make clear why it is important to keep them consistent.